packages/ui/src/components/Label/Label.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactText } from 'react';
+import { FormLabel, forwardRef, HTMLChakraProps } from '@chakra-ui/react';
+
+export type LabelSizeProps = 'sm' | 'md' | 'lg';
+
+export interface LabelProps extends HTMLChakraProps<'label'> {
+  children: ReactText | ReactText[];
+  size?: LabelSizeProps;
+  required?: boolean;
+}
+
+export const Label = forwardRef<LabelProps, 'label'>(({ size = 'sm', required, children, ...props }, ref) => {
+  if (required) {
+    return (
+      <FormLabel ref={ref} color="gray.700" fontSize={size} required={required} {...props}>
+        {children} <span style={{ color: 'red' }}>*</span>
+      </FormLabel>
+    );
+  }
+  return (
+    <FormLabel ref={ref} color="gray.700" fontSize={size} required={required} {...props}>
+      {children}
+    </FormLabel>
+  );
```

Comment:
Maybe we could reduce it to one return. Something like:
`
{children}
{required ? <span style={{ color: 'red' }}>*</span> : ''}
`